.github/workflows/ci.yml

```diff
@@ -11,46 +11,66 @@ jobs:
     name: build
 
     runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        #img: [ "centos:7", "centos:8", "rockylinux:8", "ubuntu:20.04", "ubuntu:22.04" ]
+        img: ["ubuntu:20.04", "ubuntu:22.04", "centos:8" ]
```

Comment:
Recommend adding to matrix `infiniband: ["On", "Off"]` and `numactl: ["On", "Off"]` to test different combinations of libraries.